product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx

```diff
@@ -6,9 +6,8 @@ navigation:
 ---
 
 If you plan on deploying a specific version of Postgres Distributed or a specific Postgres distribution or version, you will need to select the appropriate images and image versions.
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
-
-1. Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
-1. [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name). 
-1. [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
+Before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
 
+1.  Identify your repository name and retrieve your user token as explained in [EDB private image registries](/postgres_distributed_for_kubernetes/latest/private_registries/).
+1.  [Identify the image names for your environment](/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name).
+1.  [Assemble your deployment command](/postgres_distributed_for_kubernetes/latest/identify_images/assemble_command).
```

Comment:
@djw-m Should these be relative paths, or does it not matter?

Reply:
@gvasquezvargas @djw-m This is the only issue still remaining on this pull request. I know when we are referring to another EDB doc set we use this nomenclature for the link. But within the same doc set, I thought we were using relative paths? Or is this okay? What's our best practice?

Reply:
Same doc set, same sections. and this is the index so you don't even need paths for most of them....

In sequence:

../private_registries
identify_image_name
assemble_command